config/routes.rb

```diff
@@ -32,11 +32,7 @@
       to: "latest_changes#index",
       as: :latest_changes
 
-  get "/topic/:topic_slug/:subtopic_slug/email-signup",
-      to: "email_signups#new",
-      as: :email_signup
-  post "/topic/:topic_slug/:subtopic_slug/email-signup",
-       to: "email_signups#create"
+  get "/topic/:topic_slug/:subtopic_slug/email-signup", to: redirect("/email-signup?topic=/topic/%{topic_slug}/%{subtopic_slug}")
```

Comment:
Can we add a comment here to explain why this redirect exists?

Reply:
Sorry yep